php/chubbyphp-roadrunner/composer.json

```diff
@@ -1,6 +1,7 @@
 {
   "require": {
-    "chubbyphp/chubbyphp-framework": "~2.8.0",
+    "chubbyphp/chubbyphp-framework": "~3.0.0",
+    "chubbyphp/chubbyphp-framework-router-fastroute": "~1.0",
```

Comment:
do we need to specify version ?

Reply:
no and fastroute is now also not required as an explicit dependency anymore

Reply:
could we then use an `*` to force latest version ?

Reply:
already did that: https://github.com/the-benchmarker/web-frameworks/pull/3037/commits/6dd75eb1100e20bf17610dc1cbcfe5be3f221b17

Reply:
Ok, UI tricks me 😚